data/tlg0551/tlg007/tlg0551.tlg007.perseus-eng2.xml

```diff
@@ -2,9 +2,9 @@
 	  <teiHeader>
 		    <fileDesc>
 			      <titleStmt>
-				        <title>Wars in Spain </title>
-				        <author n="Appian">Appian</author>
-				        <editor role="editor">Horace White</editor>
+				        <title>Wars in Spain</title>
```

Comment:
here the title is the English title. —
so the file title should always match the label for the edition or translation in the its file

Reply:
should be Ιβηρική